fares-data-build-tool/create-data
# fix: cfd-452 include companion in error message

# Description

Include companion in error message on `/managePassengerGroup`

# Testing instructions

Try combinations of missing, negative and large than group limit values to see various error messages

# Type of change
